test/command/suite/load/index/offline/vector/text_without_section.expected

```diff
@@ -11,7 +11,7 @@ load --table Docs
 [[0,0.0,0.0],1]
 table_create Words TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
 [[0,0.0,0.0],true]
-column_create Words docs_sentences COLUMN_INDEX|WITH_POSITION Docs sentences
+column_create Words docs_sentences COLUMN_INDEX Docs sentences
```

Comment:
It uses `TokenBigram`. So it's an index for full text search.

Reply:
Oh...
In that case, these tests should be an error.

I'll fix them and commit to #850.
Because these fixes are related #850's modify.
So, I close this pull request.